feature-libs/my-account/organization/src/components/shared/organization-list/organization-list.service.ts

```diff
@@ -87,7 +102,10 @@ export abstract class OrganizationListService<T, P = PaginationModel> {
    */
   protected getStructure(): Observable<TableStructure> {
     return combineLatest([
-      this.tableService.buildStructure(this.tableType),
+      this.tableService.buildStructure(
+        this.tableType,
+        this.defaultTableStructure
```

Comment:
We are not doing deepMerge here, is it OK?

Reply:
It's ok for now. Might be improved over time